api/src/main/java/org/openmrs/module/ModuleFactory.java

```diff
@@ -194,7 +194,8 @@ public static void loadModules(List<File> modulesToLoad) {
 					try {
 						Module mod = loadModule(f, true); // last module loaded wins
 						log.debug("Loaded module: " + mod + " successfully");
-					} catch (Exception e) {
+					}
```

Comment:
do i understand right that you want to have closing brace, catch and opening brace on one line?
the code formatter changes that to 
closing brace on one line and 
catch and opening brace on the next line